components/api/db/cellery_hub_db.bal

```diff
@@ -41,10 +41,13 @@ public function getOrganization(string orgName, string userId) returns json | er
     json resPayload = null;
     if counter == 1 {
         log:printDebug(io:sprintf("Building the response payload for getOrganization. user : %s, orgName : %s", userId, orgName));
-        resPayload.description = encoding:byteArrayToString(orgRes.description);
+        resPayload.description = "";
+        if (!(orgRes.description is ())) {
+            resPayload.description = encoding:byteArrayToString(<byte[]>orgRes.description);
+        }
         resPayload.summary = orgRes.summary;
         resPayload.websiteUrl = orgRes.websiteUrl;
-        resPayload.author = orgRes.firstAuthor;
+        resPayload.firstAuthor = orgRes.firstAuthor;
```

Comment:
Would changing resPayload.author to resPayload.firstAuthor change the output of the API ? If so wouldn't this break the UI ?

Reply:
No. Actually, it fixes an issue in API